BraintreeCore/src/test/java/com/braintreepayments/api/BraintreeHttpClientUnitTest.java

```diff
@@ -251,7 +280,22 @@ public void postAsync_withNullConfiguration_andRelativeUrl_postsCallbackError()
     }
 
     @Test
-    public void post_withPathAndDataAndCallback_whenClientTokenAuthAndInvalidJSONPayload_postsCallbackError() throws InvalidArgumentException {
+    public void postAsync_withNullConfigurationAndAbsoluteURL_doesNotSetABaseURLOnTheRequest() throws Exception {
```

Comment:
```suggestion
    public void postAsync_withNullConfiguration_andAbsoluteURL_doesNotSetABaseURLOnTheRequest() throws Exception {
```

Nitpick - make the test name format match the one above